code/game/gamemodes/changeling/powers/fleshmend.dm

```diff
@@ -17,7 +20,8 @@
 
 /obj/effect/proc_holder/changeling/fleshmend/process()
 	if(recent_uses > 1)
-		recent_uses--
+		recent_uses -= (1 / healing_ticks)
+		recent_uses = max(recent_uses, 1)

```

Comment:
Make this one line with this quick, easy trick; recent_uses = max(recent_uses - (1 / healing_ticks), 1)